jenkins/slave.sls

```diff
@@ -0,0 +1,60 @@
+{% from "jenkins/map.jinja" import jenkins, deploy with context %}
+
+include:
+  - .sudoersd
+
+jenkins_deps:
```

Comment:
I'm not sure where this came from and I suspect It was there before my changes - will investigerate.